docs/source/data/fetching-data.mdx

```diff
@@ -163,7 +163,7 @@ const server = new ApolloServer<ContextValue>({
 });
 
 const { url } = await startStandaloneServer(server, {
-  context: async () => {
+  context: async (req) => {
```

Comment:
I see that our example is wrong, but is this correct? Doesn't `context` take `({ req })` rather than `( req )`?

Reply:
Indeed, should be:
```suggestion
  context: async ({ req }) => {
```